### PR TITLE
Upgrade prebid to 3.16.0

### PR DESCRIPTION
**Pull request by janua**
## What does this change?

This updates Prebid.js to version 3.16.0. Prebid.js update [here](https://github.com/guardian/Prebid.js/pull/106)

@guardian/commercial-dev

### CODE REVIEW

**Review by ripecosta — APPROVED** [review]
👍

**Comment by janua** [comment]
I will wait until tomorrow morning to merge this.

**Pull request merged**
